packages/origin-247-certificate/test/onchain-certificate/onchain-certificate.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import {
+    IIssueCommandParams,
+    ONCHAIN_CERTIFICATE_SERVICE_TOKEN,
+    OnChainCertificateForUnitTestsModule,
+    OnChainCertificateService
+} from '../../src';
+
+describe('OnchainCertificateService', () => {
+    let app: TestingModule;
+    let onChainCertificateService: OnChainCertificateService;
+
+    beforeEach(async () => {
+        app = await Test.createTestingModule({
+            imports: [OnChainCertificateForUnitTestsModule]
+        }).compile();
+
+        onChainCertificateService = await app.resolve(ONCHAIN_CERTIFICATE_SERVICE_TOKEN);
+
+        await app.init();
+    });
```

Comment:
I'd move to before instead of beforeEach.

Reply:
unit tests should use `beforeEach` to reset app each time